.gitignore

```diff
@@ -114,3 +114,6 @@ nbproject/*
 *.gz
 *.sql
 *.zip
+
+# Ignore cloudflare CMI export file because it includes API keys.
+config/default/cloudflare.settings.yml
```

Comment:
No action required.  This is a very minor nitpick: I would break this out into a different commit since it represents a distinct concept from the rest of the commit.